eqdsk/file.py

```diff
@@ -143,6 +138,50 @@ def __post_init__(self):
             self.psinorm = _derive_psinorm(self.fpol)
         self._cocos = None
 
+    def __repr__(self) -> str:  # noqa: D105
+        if self.qpsi is None:
+            conv_p = ", ".join(
+                str(c.index) for c in identify_eqdsk(self, qpsi_positive=True)
+            )
+            conv_n = ", ".join(
+                str(c.index) for c in identify_eqdsk(self, qpsi_positive=False)
+            )
+            conventions_str = f"{conv_p} (+ve qpsi) - {conv_n} (-ve qpsi)"
+        else:
+            conventions_str = ", ".join(str(c.index) for c in identify_eqdsk(self))
```

Comment:
What happens here if the eqdsk has not been identified on init? Does this do the identify step that has been avoided before?